server/test/controllers/FileControllerTest.java

```diff
@@ -154,7 +159,9 @@ public void acledAdminShow_differentFileKey_returnsNotFound() {
   }
 
   @Test
-  public void acledAdminShow_globalAdmin_returnsUnauthorizedResult() {
+  public void acledAdminShow_globalAdminWithoutPrivledges_returnsUnauthorizedResult() {
+    when(mockSettingsManifest.getAllowCiviformAdminAccessPrograms(request)).thenReturn(true);
```

Comment:
(question) If the `AllowCiviformAdminAccessPrograms` setting is returning `true` and the user is a global admin (line 166) why is the status `unauthorized`?

Reply:
Wow good catch! I had written this wrong but fixing it made me realize it wasn't actually using settings manifest. Added a method to WithMockedProfiles to get this working, but let me know if you think it would be better to do this another way.

Reply:
This is great! 

We absolutely should be mocking the settings to be what we expect them to be and not the config file. Thanks!!